<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>passwordscreen</class>
 <widget class="QWidget" name="passwordscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="insulinBar">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iobDecorButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="settingsLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iobDectorButton2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="screenLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock Screen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bolusButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QLabel" name="insulinLevelLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Insulin Levels</string>
   </property>
  </widget>
  <widget class="QLabel" name="bolusLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bolus </string>
   </property>
  </widget>
  <widget class="QLabel" name="iobTime">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobTimeLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Remaining</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>INSULIN ON BOARD (IOB)</string>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Battery</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="batteryBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(87, 227, 137);</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnits2">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>340</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hrs</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnits">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="iobUnitsLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Units</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bolusButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 240, 107);
font: 75 15pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
  <widget class="QLabel" name="bolusLabel_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>0</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alert</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>231</width>
     <height>166</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pinButton7">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pinButton0">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_PlusEquals">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pinButton1">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pinButton8">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pinButton6">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pinButton3">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pinButton5">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pinButton9">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pinButton4">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pinBackButton">
      <property name="styleSheet">
       <string notr="true">font: 75 15pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pinButton2">
      <property name="styleSheet">
       <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirmPinCheckBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(98, 160, 234);
</string>
   </property>
   <property name="text">
    <string>✓</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="pinDisplayBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
